tests/Owin.Security.OpenIdConnect.Server.Tests/OAuth2AuthorizationClientCredentialsGrantTests.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Net;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Owin.Security.OpenIdConnect.Server.Tests
+{
```

Comment:
Will do, if you use R# we can use the same settings to keep formatting consistent